content/pytorch/concepts/tensors/terms/sum/sum.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.sum()'
+Description: 'Returns a tensor of a desired size, filled with random values generated from a uniform distribution in the range [0,1).'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
```

Comment:
```suggestion
---
Title: '.sum()'
Description: 'Computes the sum of all elements in a tensor.'
Subjects:
  - 'AI'
  - 'Data Science'
Tags:
  - 'AI'
  - 'Deep Learning'
  - 'Functions'
  - 'Machine Learning'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'py-torch-for-classification'
---
```

Reply:
Keep the description small and to the point. Additional stuff is added in the introductory paragraph below